package.json

```diff
@@ -5,8 +5,9 @@
   "scripts": {
     "pretest": "mvn clean package",
     "test": "karma start angularjs-portal-home/target/web/karma.conf.js  --single-run",
-  	"prestart": "mvn clean package",
-  	"start": "mvn jetty:run"
+    "posttest": "remark .",
```

Comment:
@thevoiceofzeke ignore setting has been configured, `lint-md` task has been added, eslint has been moved to `lint-js`.